src/app/components/team/Statuses/StatusListItem.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import IconButton from '@material-ui/core/IconButton';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemSecondaryAction from '@material-ui/core/ListItemSecondaryAction';
+import ListItemText from '@material-ui/core/ListItemText';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import IconMoreVert from '@material-ui/icons/MoreVert';
+import styled from 'styled-components';
+import globalStrings from '../../../globalStrings';
+import { subheading2 } from '../../../styles/js/shared';
+
+const StyledStatusLabel = styled.span`
+  color: ${props => props.color};
+  font: ${subheading2};
+  font-weight: 500;
+`;
```

Comment:
Please try to use `makeStyles` + Material UI components (like `<Typography />` with the relevant variation) instead of styled components.